hack/bootstrap-cluster.sh

```diff
@@ -46,6 +46,11 @@ kubectl apply -f $ROOT/argo-cd-apps/app-of-apps/all-applications-staging.yaml
 
 ARGO_CD_URL="https://$(kubectl get route/openshift-gitops-server -n openshift-gitops -o template --template={{.spec.host}})"
 
+echo
+echo "Configuring service account for Tekton Chains:"
+oc new-project tekton-chains
+oc adm policy add-scc-to-user anyuid -z tekton-chains-controller
```

Comment:
I was wondering if this could be done with a yaml file via the GitOps component rather than doing it here, but I couldn't figure out how.

Reply:
is that needed? I was able run it without this command, but unfortunatelly it was month ago, so do not remember details.